modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -62,6 +62,11 @@ spec:
       statsPort: 1936
 ----
 ====
++
+[NOTE]
+====
+On {rh-openstack-first} platform `LoadBalancerService` `endpointPublishingStrategy` is only supported when cloud provider is configured to create health monitors. Please note that as of {rh-openstack} 16.1 and 16.2 this is only possible with Amphora Octavia provider. Please find more information in "Setting cloud provider options" section of OpenStack installation instructions.
```

Comment:
```suggestion
On {rh-openstack-first} load balancer services, the `endpointPublishingStrategy` strategy is only supported when a cloud provider is configured to create health monitors. For {rh-openstack} 16.1 and 16.2, strategy is only possible if using the Amphora Octavia provider. 

For more information, see the "Setting cloud provider options"  section in the {rh-openstack} installation documentation.
```
@dulek

Reply:
I modified your suggestion, `LoadBalancerService` is the value of `endpointPublishingStrategy` field.